geode-protobuf/src/test/java/org/apache/geode/internal/protocol/protobuf/v1/operations/GetAllRequestOperationHandlerJUnitTest.java

```diff
@@ -69,6 +74,34 @@ public void setUp() throws Exception {
     operationHandler = new GetAllRequestOperationHandler();
   }
 
+  @Test
+  public void processReturnsErrorUnableToDecodeRequest() throws Exception {
+    Exception exception = new DecodingException("error finding codec for type");
+    ProtobufSerializationService serializationServiceStub =
+        mock(ProtobufSerializationService.class);
+    when(serializationServiceStub.decode(any())).thenThrow(exception);
+
+    BasicTypes.EncodedValue encodedKey = BasicTypes.EncodedValue.newBuilder()
+        .setJsonObjectResult("{\"someKey\":\"someValue\"}").build();
+    Set<BasicTypes.EncodedValue> keys = Collections.<BasicTypes.EncodedValue>singleton(encodedKey);
```

Comment:
It might be useful to make this test use one good key and one bad key, just to verify that we handle the good key correctly while failing the other.